.gitignore

```diff
@@ -243,3 +243,4 @@ define_sanity_output.txt
 # ezdb
 /db/
 /config/ezdb.txt
+tgstation.dme
```

Comment:
can't do this

Reply:
Yeah that was my bad, clicked the wrong thing in my program